diff --git a/setup.py b/setup.py
index 8a5dd60..ee30d53 100644
--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@ setup(
     author_email='progrium@gmail.com',
     description='Magic pipe in the sky',
     packages=find_packages(),
-    install_requires=['pyzmq-static', 'argparse', 'requests', 'colorama'],
+    install_requires=['pyzmq', 'argparse', 'requests', 'colorama'],
     zip_safe=False,
     package_data={
         'skypipe': ['satellite/*']},
