--- Makefile.orig	2013-04-08 15:10:10.809023746 -0700
+++ Makefile	2013-04-08 15:13:20.026015310 -0700
@@ -108,7 +108,7 @@
 
 gen-py.go: py.c utils.c callbacks.c types.h
 	@$(ECHO) "  DUMP-GO-SPEC\t$@"
-	$(GCC) -S -o /dev/null -fdump-go-spec=$@ $< || (rm -f $@; false)
+	$(GCC) -S -o /dev/null -fdump-go-spec=$@ $(FFI) $< || (rm -f $@; false)
 
 types.c types.h types.sym: types.in types
 	@$(ECHO) "  TYPES\t\t$@"
