diff --git a/src/mplayerprocess.cpp b/src/mplayerprocess.cpp
index 3060b48..0821e72 100644
--- a/src/mplayerprocess.cpp
+++ b/src/mplayerprocess.cpp
@@ -513,7 +513,7 @@ void MplayerProcess::parseLine(QByteArray ba)
                             if (rx_mkvchapters_timestamp.indexIn(line) > -1) {
                                 int id = rx_mkvchapters_timestamp.cap(1).toInt();
                                 int64_t timestamp = rx_mkvchapters_timestamp.cap(2).toLongLong();
-                                qDebug("MplayerProcess::parseLine: mkv chapters: ID %d, START %"PRId64, id, timestamp);
+                                qDebug("MplayerProcess::parseLine: mkv chapters: ID %d, START %" PRId64, id, timestamp);

                                 if (!md.chapters_timestamp.contains(id))
                                     md.chapters_timestamp.insert(id, timestamp);
diff --git a/src/myprocess.cpp b/src/myprocess.cpp
index e365a9f..4b68080 100644
--- a/src/myprocess.cpp
+++ b/src/myprocess.cpp
@@ -172,7 +172,7 @@ void MyProcess::procFinished()
     qDebug("MyProcess::procFinished");

 #if !USE_TEMP_FILE
-    qDebug("MyProcess::procFinished: Bytes available: %"PRId64, (int64_t)bytesAvailable());
+    qDebug("MyProcess::procFinished: Bytes available: %" PRId64, (int64_t)bytesAvailable());

     if (bytesAvailable() > 0) readStdOut();
